.github/workflows/linter.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: super-linter/super-linter@v6.3.0
+        uses: super-linter/super-linter@v6.5.1
```

Comment:
Looking at master it looks like this is already done: https://github.com/influxdata/telegraf/blob/master/.github/workflows/linter.yml#L57

Reply:
You are right here! No action needed here.